cmd/consumer/main_test.go

```diff
@@ -77,7 +87,9 @@ func TestConsumeEvent(t *testing.T) {
 			data.ID = "123"
 			data.ReqURL = test.reqURL
 			data.ReqMethod = test.method
-
+			if data.ReqMethod == "POST" {
```

Comment:
```suggestion
			if data.ReqMethod == http.MethodPost {
```

Reply:
this was also in a number of other places -- fixed, thanks.